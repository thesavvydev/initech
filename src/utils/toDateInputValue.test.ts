import { describe, expect, test } from "vitest";
import toDateInputValue from "./toDateInputValue";

describe("toDateInputValue", () => {
  test("matches snapshot", () => {
    const result = toDateInputValue(new Date());
    expect(result).toMatchSnapshot();
  });

  test("returns input ready date format", () => {
    const result = toDateInputValue(new Date(2024, 10, 18));
    expect(result).toBe("2024-11-18");
  });
});
