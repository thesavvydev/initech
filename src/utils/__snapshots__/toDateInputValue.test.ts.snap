// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`toDateInputValue > matches snapshot 1`] = `"2024-11-18"`;
